server/src/dtos/tag.dto.ts

```diff
@@ -55,6 +56,7 @@ export class TagResponseDto {
 export function mapTag(entity: TagEntity): TagResponseDto {
   return {
     id: entity.id,
+    parentId: entity.parentId ?? undefined,
     name: entity.value.split('/').at(-1) as string,
     value: entity.value,
```

Comment:
We could, sure. But the API creates a tag with `name` so it would be a weird API to not return it in the response. I think semantics are more important than the extra data optimizations in this case.

Reply:
Sure, that's fine too. I don't think this is used in a bulk response context so the difference is negligible.

Reply:
This mapper is used everywhere including get all tags. I think returning 1k records is still very little data and it seems unlikely a user would have over 1k tags, so I can't imagine this being an actual issue.

Reply:
I was thinking more about whether assets outside of the asset viewer have this in the response. I agree it doesn't really matter for listing all the tags.

Reply:
I think this _is_ included by default in most asset responses, including the timeline response. This is a bigger issue that I hope to look into once the kysely migration is done.